pages/18f/projects-partners/leading-projects.md

```diff
@@ -133,10 +134,11 @@ happening.
 - Ensure that
   [project impact](https://docs.google.com/document/d/1XHYJ8MTauUnwX7pDwcb8Pw0H2CftRDekJFa0CV6kanY/edit?usp=sharing)
   metrics are gathered throughout the project
+- Track and mitigate risks throughout the project, using the [risk tracker template](https://docs.google.com/spreadsheets/d/1KHX8dSPvYQ9w4Dy4Y2K50TkHRDA9H8Z76p001ulJuJY). ([Download template here.]({% download "downloads/TEMPLATE_Risk_tracker.xlsx" %}))
```

Comment:
The handbook [guides us to avoid](https://handbook.tts.gsa.gov/updating-the-handbook/#disclaimers-and-cautions) "click here" type of links.

I think we should use the same pattern of linking for all 5 of these templates. Maybe like this:

- [risk tracker template]() ([download]())
- [weekly ship]() ([download]())
- etc

Eliminate the "Non-18F folks may be interested in" bits

Reply:
@jskinne3 good suggestion, implemented